CG/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Kevin Squire <kevin@secondspectrum.com> and contributors
```

Comment:
```suggestion
Copyright (c) 2021 Kevin Squire <kevin@secondspectrum.com>, Simon Byrne <simonbyrne@gmail.com> and contributors
```

Reply:
@kmsquire Actually, there  is a license in the root directory: why 2 separate ones?

Reply:
This one was created by `PkgTemplates.jl` (see https://github.com/invenia/PkgTemplates.jl/pull/299).  I've removed this one int he latest push.